src/pytorch_lightning/loggers/wandb.py

```diff
@@ -339,6 +339,9 @@ def __init__(
         elif dir is not None:
             dir = os.fspath(dir)
 
+        if project == "lightning_logs" and "WANDB_PROJECT" in os.environ:
+            project = os.environ["WANDB_PROJECT"]
```

Comment:
I think it would be safer to make `project` None by default, and then have

```suggestion
        if project is None:
            project = os.environ.get("WANDB_PROJECT", "lightning_logs")
```

Reply:
@carmocca That's how it was being done initially but adding the default project was added recently
cc https://github.com/Lightning-AI/lightning/issues/16028#issuecomment-1356148429 @awaelchli

Reply:
Fine either way as long as there are no regressions. As long as we can have `WandbLogger()` default to "lightning_logs" for consistency, that's fine.

Reply:
updated